v2/internal/attacktechniques/aws/persistence/iam-create-backdoor-role/malicious_policy.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "ec2.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole"
+    },
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "arn:aws:iam::193672423079:root"
```

Comment:
this makes me think of #145, what AWS account ID should we be using here? 

do you think we should add a role inline policy to explicit deny `*` to avoid unnecessary exposure?

Reply:
added a permissions boundary in [c3fcba6](https://github.com/DataDog/stratus-red-team/pull/478/commits/c3fcba62186d0c1ba74eee44d9ad67f98bca373b). Was simpler than an inline policy, because we can reuse the CreateRole call and don't need to make another API call